src/common/AssemblyResolver.cs

```diff
@@ -0,0 +1,108 @@
+﻿// Copyright (c) Microsoft. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+
+using System;
+using System.Diagnostics;
+using System.IO;
+using System.Reflection;
+
+namespace Microsoft.DotNet.Build.Tasks.Packaging
```

Comment:
I think the namespace should change, perhaps Microsoft.DotNet.Build.Common.Desktop?  I include "Desktop" in the namespace as presumably this class is only necessary when running on desktop CLR?